Volha_Ivanova/Module_1_Java/Classes_and_Objects/src/com/epam/cdp/volha_ivanova/task2/PerformanceArrayLinkedLists.java

```diff
@@ -0,0 +1,103 @@
+package com.epam.cdp.volha_ivanova.task2;
+
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.Random;
+
+public class PerformanceArrayLinkedLists {
+	/**
+	 * @author Volha_Ivanova Methods for determining time for add, get and
+	 *         remove object in ArrayList and LinkedList
+	 * @param MAX
+	 *            Maximum value of elements in the collection
+	 */
+	public static void performanceAddArrayLinkedLists(int MAX) {
+		long addTimeArrayList = 0;
+		long addTimeLinkedList = 0;
+		addTimeArrayList = System.nanoTime();
+		ArrayList<Integer> arrayList = new ArrayList<Integer>();
```

Comment:
You can split the method to more specific.
For example logic of adding and calculate addTime is the same for lists. So you can have method 

```
private long calculateAddPerformance(List<Integer> list)
{
addTimeList = System.nanoTime();
for (int i = 0; i < MAX; i++) {
Integer nextInt = new Random().nextInt();
list.add(nextInt);
}
return System.nanoTime() - addTimeList;
}
```

but in basic method you should use the method with specific  implementation
for example

int linkedListTime = calculateAddPersormance(new LinkedList<Integer>());

This comment applies for all another collection's evaluations and another method (get, put...)